packages/gatsby/.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+    [
+      "../../.babelrc.js",
+      {
+        "browser": true
+      }
+    ]
+  ]
+}
```

Comment:
this is correct, you shouldn't need the babelrc here, the existing setup will handle it correctly when it's imported in the browser code.